datavec/datavec-data/datavec-data-image/src/test/java/org/datavec/image/loader/TestNativeImageLoader.java

```diff
@@ -175,6 +176,22 @@ public void testConvertPix() throws Exception {
         assertEquals(depth1, array9.size(2));
         assertEquals(h5, array9.size(3));
         assertEquals(w5, array9.size(4));
+
```

Comment:
This test is not passing. Simplely because I don't know how would you describe an image with 5 channels in INDArray.

Reply:
Channel option removed, this is no longer an issue. Closing this.